substrafl/strategies/fed_pca.py

```diff
@@ -70,12 +73,20 @@ class FedPCA(Strategy):
             the highest eigen-values.
     """
 
-    def __init__(self, algo: Algo):
+    def __init__(
+        self,
+        algo: Algo,
+        metric_functions: Optional[Union[Dict[str, Callable], List[Callable], Callable]] = None,
```

Comment:
does metrics make sense in the context of fedPCA? 🤨

Reply:
I was asking myself the same question, but we are using them in test/algorithms/pytorch/test_fed_pcq.py, so I assumed yes 😇